apps/condo/pages/auth/register.tsx

```diff
@@ -385,9 +386,18 @@ const ValidatePhoneForm = ({ onFinish, onReset }): React.ReactElement<IValidateP
             variables,
             intl,
             form,
-            ErrorToFormFieldMsgMapping,
+            OnErrorMsg: null,
         }).catch(error => {
-            console.error(error)
+            if (has(error, 'graphQLErrors')) {
+                const smsTimeoutError = error.graphQLErrors.find((err) => (err.name.indexOf(TOO_MANY_REQUESTS) === 0))
+                if (smsTimeoutError) {
+                    const { data: { timeRemain = 0 } } = smsTimeoutError
+                    notification.error({
+                        message: SMSTooManyRequestsError,
+                        description: intl.formatMessage({ id: 'pages.auth.register.error.smstimeout' }, { timeRemain }),
+                    })
+                }
+            }
```

Comment:
If another kind of error occurs on server, nothing will be displayed to user, because you have suppressed default errors.